### ISSUE TITLE
1.16.0 / 1.16.1 / 1.17.0 : No matching export in "node_modules/esbuild-plugin-polyfill-node/polyfills/empty.js" for import "writeFile"

### ISSUE THREAD

**Issue by raskyer** [remix-run/remix#6557]
### What version of Remix are you using?

1.17.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

- Install Remix 1.17.0
- Install `ical-generator`
- Run `npm run dev`

(On the history dev server but I suspect the same effect on new dev server)

### Expected Behavior

Works as in 1.15.0

### Actual Behavior

✘ [ERROR] No matching export in "node_modules/esbuild-plugin-polyfill-node/polyfills/empty.js" for import "writeFile"

    node_modules/ical-generator/dist/index.js:60:13:
      60 │ `,t}};import{writeFile as H,writeFileSync as W,promises as q}from"...
         ╵              ~~~~~~~~~

✘ [ERROR] No matching export in "node_modules/esbuild-plugin-polyfill-node/polyfills/empty.js" for import "writeFileSync"

    node_modules/ical-generator/dist/index.js:60:28:
      60 │ `,t}};import{writeFile as H,writeFileSync as W,promises as q}from"...
         ╵                             ~~~~~~~~~~~~~

✘ [ERROR] No matching export in "node_modules/esbuild-plugin-polyfill-node/polyfills/empty.js" for import "promises"

    node_modules/ical-generator/dist/index.js:60:47:
      60 │ ...ile as H,writeFileSync as W,promises as q}from"fs";var F=(g=>(g...

**Comment by brophdawg11** [comment]
This might be handled by https://github.com/remix-run/remix/pull/6547

**Comment by MichaelDeBoey** [comment]
We just published version `1.17.1-pre.0` which includes #6562 (a possible fix). If you'd like to take it for a test run please try it out and let us know what you think!

**Comment by huyb1991** [comment]
Tested `1.17.1-pre.0`, works for me following syntax:

```
import { promises } from 'fs';

return promises.writeFile(FILE_PATH, content);
```

**Comment by MichaelDeBoey** [comment]
Closed by #6562
